cli/script/command-executor.ts

```diff
@@ -59,8 +62,12 @@ export interface PackageWithMetrics {
     metrics?: UpdateMetricsWithTotalActive;
 }
 
-export var sdk: AccountManager;
 export var log = (message: string | Chalk.ChalkChain): void => console.log(message);
+export var sdk: AccountManager;
+export var spawn = childProcess.spawn;
+export var getTmpDir = (): string => {
```

Comment:
I almost think it would be clearer to just use "os.tmpdir()" below instead of wrapping that expression in a method. If you wanted just a method, could t you do something like "var {tmpdir} = os"?